bitnami/solr/templates/service-account.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
The service account should only be created if `.Values.serviceAccount.create` is set to true.

Reply:
added